borg/helpers.py

```diff
@@ -237,10 +237,17 @@ def parse_timestamp(timestamp):
 
 def load_excludes(fh):
     """Load and parse exclude patterns from file object. Empty lines and lines starting with '#' are ignored, but
-    whitespace is not stripped.
+    whitespace is not stripped. The pattern style is determined using a line prefix.
     """
-    patterns = (line.rstrip('\r\n') for line in fh if not line.startswith('#'))
-    return [ExcludePattern(pattern) for pattern in patterns if pattern]
+    result = []
+
+    for pattern in (line.rstrip('\r\n') for line in fh if not line.startswith('#')):
+        if pattern and pattern[:4].upper() == "{RE}":
+            result.append(ExcludeRegex(pattern[4:]))
+        elif pattern:
```

Comment:
if we decide to go for requiring a prefix for glob later, we should allow it already.